exonize/sqlite_utils.py

```diff
@@ -645,68 +647,81 @@ def create_filtered_full_length_events_view(db_path: str, timeout_db: int) -> No
         cursor = db.cursor()
         cursor.execute("""
         CREATE VIEW IF NOT EXISTS Filtered_full_length_events AS
-        WITH filtered_overlapping_queries AS (
-        WITH overlaping_fragments AS (
-        WITH gene_id_counts AS (
-        SELECT f.gene_id, f.CDS_start, f.CDS_end
-        FROM Fragments AS f
-        WHERE f.percent_query >= 0.9
-        GROUP BY f.gene_id, f.CDS_start, f.CDS_end
-        HAVING COUNT(*) > 1)
+        WITH
+        -- Identify candidate fragments that satisfy our coverage and in-frame criteria.
```

Comment:
@arvestad Could you please check this view. Thanks!

Reply:
This change is too big for me to be able to review within GitHub. I will accept the PR and look in the source.